eng/Versions.props

```diff
@@ -13,11 +13,15 @@
   <!-- Production Dependencies -->
```

Comment:
The ask in the past has been to do the branding changes at the same time as the merge from internal so update the VersionPrefix as well.  See PR 4690